### PR TITLE
Merge Network-2018-02-01 to Master Networking Swagger Changes for Feb 2018

### PR DESCRIPTION
**Pull request by dedhar**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by murilogr** [comment]
Do not merge until https://github.com/Azure/azure-rest-api-specs/pull/2567 is merged into Network-2018-02-01

**Review by murilogr — CHANGES_REQUESTED** [review]
Do not merge until https://github.com/Azure/azure-rest-api-specs/pull/2567 is merged into Network-2018-02-01

**Review by murilogr — COMMENTED** [review]
https://github.com/Azure/azure-rest-api-specs/pull/2567 was merged. It contains all my changes. Looks good for my changes.

**Comment by murilogr** [comment]
It depends on the fix https://github.com/Azure/azure-rest-api-specs/pull/2646

**Comment by dsgouda** [comment]
@dedhar can you pull down the latest changes from master into `Network-2018-02-01` and push the commit. That might probably be causing the build errors

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/virtualNetwork.json [hunk 1]
```diff
@@ -0,0 +1,1258 @@
...
+          "default": false,
+          "description": "Indicates if DDoS protection is enabled for all the protected resources in the virtual network. It requires a DDoS protection plan associated with the resource."
+        },
+        "enableVmProtection": {
+          "type": "boolean",
+          "default": false,
+          "description": "Indicates if VM protection is enabled for all the subnets in the virtual network."
+        },
+        "ddosProtectionPlan": {
+          "$ref": "#/definitions/SubResource",
```
Comment (dsgouda) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/virtualNetwork.json]: Please fix this reference to `./network.json#/definitions/SubResource`

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/virtualNetwork.json (hunk 1 above)
Comment (dedhar) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-02-01/virtualNetwork.json]: @murilogr : Can you take a look and fix this in your original PR?

**Review by murilogr — APPROVED** [review]

**Comment by dsgouda** [comment]
@lmazuel Is the python failure due to `XmsParameterLocation` errors reported?

**Comment by dsgouda** [comment]
Do not merge before https://github.com/Azure/azure-rest-api-specs/pull/2644

**Comment by dsgouda** [comment]
@vinayada1  Please pull the latest changes from this branch and work on your changes

**Comment by dsgouda** [comment]
@lmazuel Can you take a look at the CI failures [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/357129361)

**Comment by dsgouda** [comment]
All builds are a go!

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by lmazuel** [comment]
@AutorestCI rebuild azure-sdk-for-python

**Pull request merged**